src/services/FilesActionHandler.js

```diff
@@ -20,36 +20,35 @@
  *
  */
 
-import { encodePath } from '@nextcloud/paths'
-
 /**
- * @param {string} name the file name
- * @param {object} context the file context
+ * @param {Node} node The file to open
+ * @param {any} view any The files view
+ * @param {string} dir the directory path
  */
-export default function(name, context) {
+export default function(node, view, dir) {
 	// replace potential leading double slashes
-	const path = `${context.dir}/${name}`.replace(/^\/\//, '/')
```

Comment:
As discussed with @max-nextcloud let's keep this and still register the old way side by side to the new one to keep public share links working until migrated to vue